src/vs/workbench/contrib/terminalContrib/chatAgentTools/browser/tools/monitoring/outputMonitor.ts

```diff
@@ -506,6 +507,13 @@ export class OutputMonitor extends Disposable implements IOutputMonitor {
 							run: async () => { }
 						}))
 					));
+					const inputDataDisposable = this._register(execution.instance.onDidInputData(() => {
```

Comment:
Does this event ignored the focus in/out events? Click in and out of the terminal when the prompt is up to double check. If not we should filter those out like we're doing here:

https://github.com/microsoft/vscode/blob/a18d41e90a83fbe551fd7db4894a54cb7d436966/src/vs/workbench/contrib/terminalContrib/chatAgentTools/browser/tools/runInTerminalTool.ts#L417-L419

Reply:
Yes, focus in /out is ignored